catalogue/webapp/components/SearchForm/SearchForm.js

```diff
@@ -57,7 +65,15 @@ const SearchForm = ({ ariaDescribedBy, compact }: Props) => {
     if (query !== inputQuery) {
       setInputQuery(query);
     }
-  }, [query]);
+
+    if (_dateFrom !== inputDateFrom) {
+      setInputDateFrom(_dateFrom);
```

Comment:
On Jenn's thing, why not just fallback to the browser date handling e.g.
`moment(new Date(_dateFrom)).format('YYYY-MM-dd')`